listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,15 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self):
+        listens_data = []
+        latest_listens = []
+        data = self.redis.get('latest_listens:{}'.format(
```

Comment:
Could you check this please?
As I'm getting AttributeError: 'list' object has no attribute 'redis'

Reply:
All the errors from Travis lead to this line